ngraph/test/type_prop/unsqueeze.cpp

```diff
@@ -37,3 +37,44 @@ TEST(type_prop, unsqueeze_dynamic)
                                                                         Dimension::dynamic(),
                                                                         Dimension::dynamic()}));
 }
+
```

Comment:
I would add a type prop test when axes input is a parameter to check that output shape is fully dynamic